projects/CENet/cenet/transforms_3d.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Sequence
+
+import numpy as np
+from mmcv.transforms import BaseTransform
+
+from mmdet3d.registry import TRANSFORMS
+
+
+@TRANSFORMS.register_module()
+class SemkittiRangeView(BaseTransform):
```

Comment:
Can this be put in mmdet3d/models/data_preprocessors/data_preprocessor.py? Refer to voxelization

Reply:
Maybe some other augmentations can be performed on the `range image` such as `range-image_shift`, so we suggest converting point cloud to range image in the `transforms` pipeline.